src/main/java/com/dreammaster/tinkersConstruct/TiCoLoader.java

```diff
@@ -1,13 +1,76 @@
 package com.dreammaster.tinkersConstruct;
 
+import static com.dreammaster.scripts.IScriptLoader.missing;
+import static gregtech.api.enums.Materials.Obsidian;
+import static gregtech.api.enums.Materials.Steel;
+import static gregtech.api.enums.Materials.Zinc;
 import static gregtech.api.enums.Mods.TinkerConstruct;
+import static gregtech.api.enums.Mods.UniversalSingularities;
+import static gregtech.api.util.GTModHandler.getModItem;
 
+import java.util.stream.Collectors;
+
+import net.minecraft.block.Block;
+import net.minecraft.init.Items;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.oredict.OreDictionary;
+
+import com.dreammaster.main.NHItems;
 import com.dreammaster.mantle.BookLoader;
+import com.dreammaster.tinkersConstruct.worldgen.ZincGravelOre;
+import com.dreammaster.tinkersConstruct.worldgen.ZincGravelOreItem;
+import com.dreammaster.tinkersConstruct.worldgen.ZincGravelWorldgen;
+import com.google.common.collect.ImmutableList;
+import com.gtnewhorizons.postea.api.BlockReplacementManager;
+import com.gtnewhorizons.postea.api.ItemStackReplacementManager;
+import com.gtnewhorizons.postea.api.TileEntityReplacementManager;
+import com.gtnewhorizons.postea.utility.BlockConversionInfo;
+
+import cpw.mods.fml.common.registry.GameRegistry;
+import gregtech.api.enums.Materials;
+import gregtech.api.enums.OrePrefixes;
+import gregtech.api.objects.MaterialStack;
+import gregtech.api.util.GTOreDictUnificator;
 
 public class TiCoLoader {
 
+    public static void overrideAlumiteMaterial() {
+        Materials.Alumite.mDefaultLocalName = "Obzinite";
+        Materials.Alumite.mMaterialList.clear();
+        Materials.Alumite.mMaterialList.addAll(
+                ImmutableList
+                        .of(new MaterialStack(Zinc, 5), new MaterialStack(Steel, 2), new MaterialStack(Obsidian, 2)));
+        Materials.Alumite.mChemicalFormula = Materials.Alumite.mMaterialList.stream().map(MaterialStack::toString)
+                .collect(Collectors.joining()).replaceAll("_", "-");
```

Comment:
We should make this change in 5u rather than here, that way we can also rename the variable and make things more clear

Reply:
Since GT5 is GTNH-only, which I forgot, it is the right place to make those changes so I will move them there.

That being said I would rather not change either the variable or the material name for a few reasons:
- Keeping the material name so that it unifies with TiCo Alumite.
- Keeping the material name so that searching for "Alumite", which most players will do, still works.
- Keeping the variable name to help in future maintenance, since other Tinker's based mods will still refer to it as Alumite and we can't change the variable name in those  since they're not GTNH-only. This material is still Tinker's Alumite, but it gets renamed in display text for GTNH reasons.